styles/docs.css

```
@@ -0,0 +1,15 @@
+@reference "./globals.css";
```

Stylelint (17.12.0)

[error] Unexpected unknown at-rule "`@reference`" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)